mono/metadata/threads.c

```diff
@@ -3722,6 +3722,11 @@ mono_thread_manage (void)
 		mono_thread_execute_interruption_void ();
 	}
 
+#ifndef ENABLE_NETCORE
```

Comment:
Is there a reason to guard this with ENABLE_NETCORE instead of DISABLE_CLEANUP?

Reply:
That name is too short, but it is a valid point -- to compose out of a finer grained set of names, vs. one omnibus one. The fine grained ifdef raises the question as to what combinations of things work or are tested. But that does not necessarily bother people -- witness autoconf.

Reply:
DISABLE_CLEANUP is already used at the other place in this commit, so it made me wonder whether all the combinations are actually necessary.

Reply:
Doesn't matter too much, DISABLE_CLEANUP is a --enable-minimal feature which can be used by anyone, while not aborting threads is  a part of netcore.